tracer/src/Datadog.Trace/ClrProfiler/AutomaticTracer.cs

```diff
@@ -42,6 +48,11 @@ void IDistributedTracer.LockSamplingPriority()
             return (SamplingPriority?)_child.TrySetSamplingPriority((int?)samplingPriority);
         }
 
+        public object GetActiveScope()
```

Comment:
Could this be renamed to `GetAutomaticActiveScope`? It's a little confusing to me that this shares the same name as `IDistributedTracer.GetActiveScope`

Reply:
Renamed to `GetAutomaticActiveScope` in 640b89411493b1bdb4f53cd5f34226569eec07d3